rinja_derive/src/config.rs

```diff
@@ -360,6 +360,26 @@ impl<'a> RawSyntax<'a> {
             }
         }
 
+        for end in [syntax.block_start, syntax.expr_start, syntax.comment_start] {
```

Comment:
Unless I'm missing something, you never check for closing delimiters here, right?